src/Tribe/Cache.php

```diff
@@ -30,6 +31,7 @@ public function hook() {
 		}
 
 		add_action( self::SCHEDULED_EVENT_DELETE_TRANSIENT, [ $this, 'delete_expired_transients' ] );
+		add_action( 'tribe_cache_flush_expired_transients', 'wp_cache_flush' );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: e40acc485779eee01c24c9fa72edd718</sub>